### PR TITLE
More changes

### PR DESCRIPTION
**Pull request by wetda3**
Really I do not get how features works I want to keep every thing of the same feature in one feature but it features don't update with the develop and its annoying.

Also camera now FINALLY, I hope scales properly

### CODE REVIEW

**Review by werydude — CHANGES_REQUESTED** [review]
Features, hotfix, etc are just fancy words for organizing code.

**tl;dr: unless it says "push" or "pull" everything is done locally**

That means, when you submit a PR to develop – a **pull** request – that gets approved, it is uploaded remotely. Additionally, since PRs are done within remote, not local, you have to **push** your local branch. The branch you're asking about can't see your local stuff if you don't!

_However_, when you start a new feature – no **push** or **pull** in the word feature! – it's done using your local copy of the base branch, in this case develop. So if you don't _update your local develop_, and subsequently update any feature branches you were working on by _rebasing_ – the word rebase doesn't have **pull** or **push** either! – then you will be using an outdated version, even if you made approved changes!

**Comment by wetda3** [comment]
I think I re-pushed this one too many times you might get a lot of messages

**Review by werydude — DISMISSED** [review]
Will fix
